rosbag2_cpp/src/rosbag2_cpp/reader.cpp

```diff
@@ -34,6 +35,18 @@ Reader::~Reader()
   reader_impl_->reset();
 }
 
+void Reader::open(const std::string & uri)
+{
+  rosbag2_cpp::StorageOptions storage_options;
+  storage_options.uri = uri;
+  storage_options.storage_id = "sqlite3";
```

Comment:
minor nit - should these 3 default values ("sqlite3", 0, 0) be defined as constants somewhere?

Reply:
the last two are directly "hardcoded" in the storage options struct - meaning I could technically leave out the following two lines of code. 
Where would you define the other constant, "sqlite3"?

Reply:
@emersonknapp I've excluded the `sqlite3` into a constant within the same file. Let me know if that works for you.

Reply:
LGTM